wildmatch.go

```diff
@@ -349,13 +358,14 @@ type token interface {
 // doubleStar is an implementation of the Token interface which greedily matches
 // one-or-more path components until a successor token.
 type doubleStar struct {
-	Until token
+	Until     token
+	EmptyPath bool
```

Comment:
My feeling on this was that implicitly the other member was “match until” and so “match empty path” seemed to correspond appropriately.  Since this will only be set at the end (hence the name of the wrapping class, `trailingComponents`), whether we can continue is equivalent to whether this matches.

So I think I'd like to leave this as it is.